src/streamlink/plugins/drdk.py

```diff
@@ -1,135 +1,64 @@
-"""Plugin for Denmark's public service channel DR (Danmarks Radio)."""
-
+import logging
 import re
 
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import validate
-from streamlink.stream import HLSStream, HDSStream
+from streamlink.stream import HLSStream
 
-LIVE_CHANNELS_API_URL = "https://www.dr.dk/mu-online/api/1.4/channel/{0}"
-VOD_API_URL = "https://www.dr.dk/mu/programcard/expanded/{0}"
+log = logging.getLogger(__name__)
 
-STREAMING_TYPES = {
-    "HDS": HDSStream.parse_manifest,
-    "HLS": HLSStream.parse_variant_playlist
-}
 
-_url_re = re.compile(r"""
-    http(s)?://(www\.)?dr.dk
-    (?:
-        /[TtVv]+/
-        (?:
-            live/(?P<channel>[^/]+)
-        )?
-        (?:
-            se/(.+/)?(?P<program>[^/&?]+)
-        )?
+class DRDK(Plugin):
+    live_api_url = (
+        'https://www.dr-massive.com/api/page?device=web_browser'
```

Comment:
instead of using such a long URL use params

https://github.com/streamlink/streamlink/blob/e67c68c6457246d6072d9e5cd2353ea1d21c3c11/src/streamlink/plugins/mediaklikk.py#L30-L35

Reply:
Thanks, that's much nicer.  It also makes it easier to test stripping out unneeded parameters, which I have also now done, reducing the number to 2 now.  I've gone with ```dict()``` rather than ```{}``` because it's quicker to format.